src/redux/actions/index.test.js

```diff
@@ -0,0 +1,52 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as actions from './index';
+import * as types from '../constants/action-types';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('actions', () => {
+  it('should change search option', () => {
```

Comment:
It's better to put all cases for each action into separate test suite. For instance, wrap with 
`describe('changeSearch'), () => { ... });`